src/arena/ArenaDateHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import { Paper, useTheme } from "@mui/material";
+import { Dispatch, SetStateAction } from "react";
+import Image from "react-bootstrap/lib/Image";
```

Comment:
As long as you are working on this code, can you please replace this `Image` component from react-bootstrap with either a generic <img> tag or something from MUI? We are close to being rid of React-Bootstrap!